src/routes/safe/components/Apps/index.tsx

```diff
@@ -43,12 +44,13 @@ const operations = {
 }
 
 function Apps({ closeModal, closeSnackbar, enqueueSnackbar, openModal }) {
-  const [appList, setAppList] = useState([])
+  const [appList, setAppList] = useState<Array<SafeApp>>([])
   const [legalDisclaimerAccepted, setLegalDisclaimerAccepted] = useState(false)
-  const [selectedApp, setSelectedApp] = useState()
+  const [selectedApp, setSelectedApp] = useState<string>()
```

Comment:
this is not a valid type for this initial value, it should be either `<string | undefined>` or `<string>('')`

Reply:
Are you sure? Typescript should infer the type

Reply:
https://codewithstyle.info/Using-React-useState-hook-with-TypeScript/

`const [pill, setPill] = useState<'red' \| 'blue'>();`

Note that the inferred type of pill is now "red" | "blue" | undefined (because this piece of state is initially empty).

Reply:
👍 didn't know about that, sorry